src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/RolesController.kt

```diff
@@ -133,7 +134,7 @@ class RolesController(
   @GetMapping("/roles")
   fun getRoles(
     @RequestParam(value = "adminTypes", required = false) adminTypes: List<AdminType>?,
-  ): List<Role> = rolesService.getRoles(adminTypes)
+  ): List<RoleDto> = rolesService.getRoles(adminTypes).map { RoleDto.fromDomain(it) }
```

Comment:
same with this one

Reply:
Discussed with @karenmillermoj and agreed OK